cms/templates/js/course-outline.underscore

```diff
@@ -70,7 +70,7 @@ if (is_proctored_exam) {
 <li class="outline-item outline-<%- xblockType %> <%- visibilityClass %> is-draggable <%- includesChildren ? 'is-collapsible' : '' %> <%- isCollapsed ? 'is-collapsed' : '' %>"
     data-parent="<%- parentInfo.get('id') %>" data-locator="<%- xblockInfo.get('id') %>">
 
-    <span class="draggable-drop-indicator draggable-drop-indicator-before"><i class="icon fa fa-caret-right"></i></span>
+    <span class="draggable-drop-indicator draggable-drop-indicator-before"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
@clrux Note that this is for reordering units, subsections, and sections on the course outline page (not related to the DnD component). There is a blue caret plus a horizontal line. We will need help understanding what to do with these.